start.js

```diff
@@ -135,7 +135,7 @@ async function runFastify (args, additionalOptions, serverOptions) {
     options = deepmerge(options, serverOptions)
   }
 
-  if (opts.options && file.options) {
+  if (opts.options || file.options) {
```

Comment:
It is absolutely a wrong fix.
The `opts.options` expected to be `true` in order to allow file options.
It is the documented behavior.